pyproject.toml

```diff
@@ -1,11 +1,48 @@
-[build-system]
-requires = [
-    "setuptools>=63",
-    "wheel"
+[project]
+name = "conventional_pre_commit"
+version = "2.1.1"
+description = "A pre-commit hook that checks commit messages for Conventional Commits formatting."
+readme = "README.md"
+license = { file = "LICENSE" }
+classifiers = ["Programming Language :: Python :: 3 :: Only"]
+keywords = ["git", "pre-commit", "conventional-commits"]
+authors = [
+    { name = "Compiler LLC", email = "dev@compiler.la" }
 ]
+requires-python = ">=3.7"
+dependencies = []
+
+[project.urls]
+code = "https://github.com/compilerla/conventional-pre-commit"
+tracker = "https://github.com/compilerla/conventional-pre-commit/issues"
+
+[project.optional-dependencies]
+dev = [
+    "black",
+    "build",
+    "flake8",
+    "pre-commit",
+    "pytest",
+    "pytest-cov"
```

Comment:
If we intended to publish to PyPI manually from within the devcontainer, then yes we would need `twine`.

But since we have a GitHub Action that does the publishing for us, we don't need it in the dev requirements.

Reply:
Ah I see. Thanks!